bin/kafka-producer-perf-test.sh

```diff
@@ -17,4 +17,4 @@
 if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
-exec $(dirname $0)/kafka-run-class.sh kafka.tools.ProducerPerformance $@
+exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance $@
```

Comment:
The new ProducerPerformance is only going to support the new producer while the old producer supports both the old and the new producer. Also, we are not removing the old ProducerPerformance class. So, one can still use it if needed. Given that, I don't feel that we need to preserve the command line options. However, the command line option for the new ProducerPerformance is weird since it doesn't use jopt. The reason is probably because that class was in clients and we didn't want to drag in the jopt dependency to clients. Now that the new ProducerPerformance is in tools, we can change the command line options to use jopt so that it's consistent with other tools.